ts/test/tunnels-test/tunnelHostAndClientTests.ts

```diff
@@ -464,24 +464,31 @@ export class TunnelHostAndClientTests {
 
     @test
     public async connectRelayClientAddPortInUse() {
```

Comment:
This test didn't work at all before; it was trying to use `socket.connect()` instead of `socket.listen()` to create a port conflict. It appeared to succeed only because it never awaited the bad code.